src/styles/1-helpers/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$white: #fff;
+$blue: #00acdc;
+$black: #060b35;
+$grey: #616070;
```

Comment:
It is better to give names so that their purpose is clear, and not just to repeat the name of the color, for example.
Because the color and other values may change during development:
```
$cardBackgroundColor: #fff;
$buttonColor: #00acdc;
etc.
```